code/modules/projectiles/guns/rifles.dm

```diff
@@ -1499,7 +1490,7 @@
 		/obj/item/ammo_magazine/rifle/chamberedrifle/flak,
 	)
 	attachable_allowed = list(
-		/obj/item/attachable/scope/unremovable/tl127,
+		/obj/item/attachable/scope/mini,
```

Comment:
you're removing all these unremoveable scope types with just the generic miniscope sprite?